api/src/com/cloud/dc/StorageNetworkIpRange.java

```diff
@@ -21,20 +21,19 @@
 import org.apache.cloudstack.api.InternalIdentity;
```

Comment:
@nitin-maharana I saw the related mail so went ahead and looked at it. You don't need my review but I am never merging blindly (unless as RM;)

Is there a reason you changed this file? I don't see any significance in the changes.

Reply:
Hi @DaanHoogland, Firstly thanks for reviewing it.

I agree this change has no significance to the feature. But exactly I don't remember why I changed the order because I implemented it long back. I guess, one reason must be to maintain all UUIDs on one side, gateway & netmask another side, startIp and endIp together in the middle.